connectivity/perf/benchmarks/netperf/perfpod.go

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package netperf
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/cilium/cilium-cli/connectivity/check"
+	"github.com/cilium/cilium-cli/connectivity/perf/common"
+	"github.com/cilium/cilium-cli/utils/features"
+)
+
+const (
+	messageSize = 1024
+	tool        = "netperf"
```

Comment:
Nit: could we rename this to `netperfToolName`? It's more descriptive.

Reply:
sure, changed.